src/tests/Microsoft.PowerFx.Core.Tests/TypeSystemTests/TypeTests.cs

```diff
@@ -73,14 +70,14 @@ public void TryGetFieldTypeLookupTestWithoutDisplayName(string inputDisplayOrLog
         [Fact]
         public void TryGetFieldTypeLookupNullTest()
         {
-            var r1 = RecordType.Empty()
-                        .Add(new NamedFormulaType("F1", FormulaType.Number, "Display1"));
-
-            Assert.Throws<ArgumentNullException>(
-                () => r1.TryGetFieldType(null, out var actualLogical, out var formulaType));
-
-            Assert.Throws<ArgumentException>(
-                () => r1.TryGetFieldType(string.Empty, out var actualLogical, out var formulaType));
+            var r1 = RecordType.Empty().Add(new NamedFormulaType("F1", FormulaType.Number, "Display1"));
+
+            // These tests are skipped when not in DEBUG configurationn
+            // as we rely on Contract.Asserts to throw

```

Comment:
These were relying on Contracts.Check, given that they were throwing in release. If thats a perf issue and they should be using Assert, that's a separate question.